tests/modules/cnvkit/batch/test.yml

```diff
@@ -0,0 +1,101 @@
+- name: cnvkit batch test_cnvkit_tumoronly
+  command: nextflow run tests/modules/cnvkit/batch -entry test_cnvkit_tumoronly -c tests/config/nextflow.config
+  tags:
+    - cnvkit
+    - cnvkit/batch
+  files:
+    - path: output/cnvkit/reference.antitarget-tmp.bed
+      md5sum: d41d8cd98f00b204e9800998ecf8427e
```

Comment:
This is the checksum for emtpy files, if I am not wrong. maybe try with the human reads?